www/js/init.js

```diff
@@ -25,9 +25,11 @@
 require.config({
     baseUrl: 'js/lib',
     paths: {
-        'jquery': 'jquery-2.1.4',
+        //'jquery': 'jquery-2.1.4', // Implements #268
+        //'jquery': 'jquery-3.2.1', // Full version
+        'jquery': 'jquery-3.2.1.slim', // Slim version
```

Comment:
Of course, we should remove the commented lines.
Please also switch to the new version of jQuery for the unit tests (it also works), in tests/init.js, and also in /nbproject/project.properties (this is for NetBeans autocompletion with requirejs : I'll do the testing)

I don't remember why we kept the version number in the file name of jQuery (and not for every other library). It might be the occasion to remove it, so that the file name would simply be jQuery.js